### PR TITLE
Translation fixes for everything but src/components

### PR DESCRIPTION
**Pull request by Kegsay**
It's regrettably nigh impossible for me to test all of these changes :(

### CODE REVIEW

File: src/autocomplete/DuckDuckGoProvider.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
+/*
```
Comment (dbkr) [inline src/autocomplete/DuckDuckGoProvider.js]: Hmm, the autocomplete stuff will actually be (c) 2016 Aviral Dasgupta - we should probably not claim copyright for ourselves when that's not the case.

Comment (Kegsay) [inline src/autocomplete/DuckDuckGoProvider.js]: So swap:
```
Copyright 2017 Vector Creations Ltd
```
with:
```
Copyright 2016 Aviral Dasgupta
```
sound okay? Or should we also put a 2017 Vector Creations along with it?

Comment (dbkr) [inline src/autocomplete/DuckDuckGoProvider.js]: Hmm, so of https://github.com/matrix-org/matrix-react-sdk/tree/develop/src/autocomplete, the 'Fix a bunch of linting errors' and 'Bulk change counterpart imports' are me, so everything apart from Autocompleter.js and Components.js need Vector 2017 headers.

Comment (Kegsay) [inline src/autocomplete/DuckDuckGoProvider.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/990/commits/66bce35918d10ca9a6babb09fa1571cf61e71908

**Comment by ara4n** [comment]
this lgtm. travis claims to be failing but the console says that the tests passed so i'm merging it.

**Pull request merged**